slang.h

```diff
@@ -605,6 +605,11 @@ extern "C"
 
         //
         SLANG_PARAMETER_CATEGORY_COUNT,
+
+
+        // DEPRECATED:
```

Comment:
Any reason to deprecate it instead of removing it completely?

Reply:
The case I'm worried about is when a Falcor user runs into a Slang bug. If I can just put together a bug-fix release really fast and then get them to update `dependencies.xml` I can usually unblock them nice and fast, without having to coordinate with you to do a full Falcor release to incorporate the new version.

That approach isn't going to work if updating their Slang version causes the user's Falcor build to break, because they have old source code that hasn't been updated for changes to the Slang API.

As a result, my current goal is that the Slang API needs to stay source (and ideally binary) compatible unless there is a *strong* reason why a break is needed.